### PR TITLE
Add birds nest

### PR DESCRIPTION
**Pull request by coolbop32**

### CODE REVIEW

File: src/tasks/minions/woodcuttingActivity.ts [hunk 1]
```diff
@@ -45,6 +46,14 @@ export default class extends Task {
 			loot[WoodcuttingPet!] = 1;
 			str += `\nYou have a funny feeling you're being followed...`;
 		}
+		// roll for a birds nest with an item in it that is not a clue
+		let nests = 0;
+		for (let i = 0; i < quantity; i++) {
+			if (roll(256)) {
+				nests++;
+				loot[5073] = nests;
```
Comment (Alexsuperfly) [inline src/tasks/minions/woodcuttingActivity.ts]: move this out of the for loop

File: src/lib/simulation/birdsNest.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+const RingTable = new LootTable()
+	.every('crushed nest')
+	.add('Gold ring', 1, 32)
+	.add('Sapphire ring', 1, 32)
+	.add('Emerald ring', 1, 16)
+	.add('Ruby ring', 1, 8)
+	.add('Diamond ring', 1, 2);
+
+// chance at a piece of the evil chicken outfit
+const EggTable = new LootTable()
```
Comment (Alexsuperfly) [inline src/lib/simulation/birdsNest.ts]: this always drops a seednest with a chance at an outfit piece

**Comment by gc** [comment]
Not a fan that its coded differently to how it works ingame, it should give the literal different kinds of nests instead of 1 nest that drops all the different kinds, when people open them, it will just open any of them as they all have the same name which is no issue

**Comment by gc** [comment]
Closed due to inactivity.

**Pull request closed without merging**